assets/src/dashboard/components/bookmark-chip/test/bookmark-chip.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import theme from '../../../theme';
+
+import BookmarkChip from '../';
+
+const wrapper = (children) => {
+  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider>);
+};
+
+describe('BookmarkChip', () => {
+  it('should render a bookmark chip', () => {
+    const { getByRole } = wrapper(<BookmarkChip />);
+
+    expect(getByRole('button')).toBeDefined();
```

Comment:
I'm unsure how feasible testing the correct SVG icon is, but it might be worth testing `isBookmarked`.

Reply:
good idea, i'll add that

Reply:
added 2 tests - to test for both false and true. rerunning builds now and then i'll retag ya. ✨